### DISCUSSION TITLE
天文學家震驚！ 詭異行星挑戰人類現有理論

### DISCUSSION THREAD

**Discussion by spermwhale464** [1101-GS4719-C/Python-Community-GS4719-C#1731] [General]
https://news.ltn.com.tw/news/world/breakingnews/4511077
又是一個千篇一律的文章，簡單來說，科學家發現一顆距離地球50光年的一顆行星，其屬性跟地球相似，所以覺得上面可能有生命體存在。有興趣的人可以參考看看。

**Comment by alvin119** [comment]
可以請問文章中哪裡有寫出「屬性跟地球相似，上面可能有生命體存在」嗎 ?
之前也有看到這篇文章，我認為文章的重點應該是在恆星跟行星的質量比例不尋常所以震驚到了科學家。
